<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paymentinfo</class>
 <widget class="QDialog" name="paymentinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Payment Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{background-color: #ccc;}

QDialog QPushButton:hover
{
background: #eee;
cursor: pointer;
  
}
QDialog
{
background: #333;
}
QDialog QPushButton {  border: 2px solid #666;border-radius: 5px;z-index:1000;}

 
          
QDialog #visiting,#label_2,#label,QLabel
{
 	color: #fff;  
} 
#columnView
{z-index: 0;  
}  
#listWidget_2
{
  background:#999; 
color: #fff;
}
#listWidget,#listWidget_2
{
background:#666; 
}
QColumnView
{
  background:#ccc; 
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  First Name </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>  Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>  Credit Card Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>  Credit Card Number  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>  Expiration Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Visa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mastercard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>American Express</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string notr="true"/>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>2015</number>
   </property>
   <property name="maximum">
    <number>2030</number>
   </property>
   <property name="value">
    <number>2015</number>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>91</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>91</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorFirstName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorLastName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorCardNum">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>listWidget_2</zorder>
  <zorder>listWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>comboBox</zorder>
  <zorder>spinBox</zorder>
  <zorder>spinBox_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>errorFirstName</zorder>
  <zorder>errorLastName</zorder>
  <zorder>errorCardNum</zorder>
  <zorder>firstName</zorder>
  <zorder>lastName</zorder>
  <zorder>cardNum</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
